pkg/kritis/admission/admission.go

```diff
@@ -61,13 +61,25 @@ var (
 // If one is not found, it validates against image security policies
 // TODO: Check for attestations
 func AdmissionReviewHandler(w http.ResponseWriter, r *http.Request) {
-	glog.Info("Starting admission review handler version %s ...", version.Commit)
+	glog.Infof("Starting admission review handler version %s ...", version.Commit)
 	pod, err := admissionConfig.retrievePod(r)
 	if err != nil {
 		glog.Error(err)
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
+
+	// Check, if the pod is already running. This handles scenarios when security
+	// policy is deployed after a pod is deployed.
+	// In this situation, the cron job will annotate the pod and call v1.CoreV1.Pod.Patch()
+	// kubectl patch command goes through admission review before patching the pod and will return error.
+	// In order to apply the labels for invalid pods, we skip admission review for already running pods.
+	if pods.IsPodRunning(*pod) {
```

Comment:
Does the cron job come through this function (AdmissionReviewHandler)?

Reply:
the cron job does not come through this function. But it call `kubectl apply patch` for adding annotations which go through admission review.
So, the labels never get applied.

Reply:
Now, that i think more about it, the admission controller will always return "true" on updates.
Kind of defeats the purpose of having this [here](https://github.com/grafeas/kritis/blob/master/kritis-charts/templates/kritis-server-webhook.yaml#L14) 

I think, this will be simpler and better once we add attestations to indicate if this pod was ever admitted.